eng/Packages.Data.props

```diff
@@ -261,41 +280,39 @@
     <PackageReference Update="Microsoft.Azure.WebJobs.Sources" Version="3.0.41" PrivateAssets="All"/>
     <PackageReference Update="Microsoft.Azure.WebJobs.Extensions.Rpc" Version="3.0.41" />
     <PackageReference Update="Microsoft.Azure.WebJobs.Host.Storage" Version="5.0.1" />
-    <PackageReference Update="Microsoft.Spatial" Version="7.5.3" />
-    <PackageReference Update="Newtonsoft.Json" Version="13.0.3" />
+    <PackageReference Update="Microsoft.Spatial" Version="$(MicrosoftSpatialVersion)" />
+    <PackageReference Update="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
   </ItemGroup>
 
   <!-- Packages intended for Extensions libraries only -->
   <ItemGroup Condition="'$(IsExtensionClientLibrary)' == 'true'">
-    <PackageReference Update="Microsoft.AspNetCore.DataProtection" Version="8.0.19" />
-    <PackageReference Update="Microsoft.AspNetCore.Http" Version="2.3.0" />
+    <PackageReference Update="Microsoft.AspNetCore.DataProtection" Version="10.0.1" />
+    <PackageReference Update="Microsoft.AspNetCore.Http" Version="$(MicrosoftAspNetCoreHttpVersion)" />
     <PackageReference Update="Microsoft.AspNetCore.Http.Connections" Version="1.2.0" />
     <PackageReference Update="Microsoft.Azure.Functions.Worker.Extensions.Abstractions" Version="1.3.0" />
-    <PackageReference Update="Microsoft.Extensions.Azure" Version="1.12.0" />
-    <PackageReference Update="Microsoft.Extensions.DependencyInjection.Abstractions" Version="8.0.2" />
-    <PackageReference Update="Microsoft.Extensions.Configuration" Version="8.0.0" />
-    <PackageReference Update="Microsoft.Extensions.Configuration.Abstractions" Version="8.0.0" />
-    <PackageReference Update="Microsoft.Extensions.Configuration.Binder" Version="8.0.2" />
-    <PackageReference Update="Microsoft.Extensions.Logging.Abstractions" Version="8.0.3" />
-    <PackageReference Update="Microsoft.Extensions.Logging" Version="8.0.1" />
-    <PackageReference Update="Microsoft.Extensions.Options" Version="8.0.2" />
-    <PackageReference Update="Microsoft.Extensions.Hosting" Version="8.0.1" />
+    <PackageReference Update="Microsoft.Extensions.Azure" Version="$(MicrosoftExtensionsAzureVersion)" />
+    <PackageReference Update="Microsoft.Extensions.Configuration" Version="$(MicrosoftExtensionsConfigurationVersion)" />
+    <PackageReference Update="Microsoft.Extensions.Configuration.Abstractions" Version="$(MicrosoftExtensionsConfigurationAbstractionsVersion)" />
+    <PackageReference Update="Microsoft.Extensions.Configuration.Binder" Version="$(MicrosoftExtensionsConfigurationBinderVersion)" />
+    <PackageReference Update="Microsoft.Extensions.Hosting" Version="$(MicrosoftExtensionsHostingVersion)" />
+    <PackageReference Update="Microsoft.Extensions.Logging" Version="$(MicrosoftExtensionsLoggingVersion)" />
+    <PackageReference Update="Microsoft.Extensions.Logging.Abstractions" Version="$(MicrosoftExtensionsLoggingAbstractionsVersion)" />
```

Comment:
Since this one is no longer exclusive to extensions packages do we need it to be in this special section?